conf/machine/lx2160ardb.conf

```diff
@@ -30,6 +30,7 @@ SERIAL_CONSOLES_CHECK ?= "${SERIAL_CONSOLES}"
 UEFI_XSPIBOOT ?= "LX2160ARDB_EFI_NORBOOT.fd"
 RCWXSPI ?= "XGGFF_PP_HHHH_RR_19_5_2/rcw_2000_700_2900_19_5_2"
 RCWSD ?= "XGGFF_PP_HHHH_RR_19_5_2/rcw_2000_700_2900_19_5_2_sd"
+RCWEMMC ?= "XGGFF_PP_HHHH_RR_19_5_2/rcw_2000_700_2900_19_5_2"
```

Comment:
Maybe a commit message could include a reference to the SOC documentation where this detail is technically described. Existing commit message just describe the obvious.

http://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines

Reply:
update commit message.

Reply:
In none of those documents is reasonably described how the rcw is built. But it says when which rcw for which board has to be used.